### ISSUE TITLE
functionality to paginate and export output with rtables only summarize_row_group contents

### ISSUE THREAD

**Issue by clarkliming** [insightsengineering/formatters#147]
sorry to raise this again and again but we have to have this to use export_as_txt with a fixed number of lines.

or, instead of using summarize_row_groups, use analyze instead in all tlg catalog entries.

example

https://insightsengineering.github.io/tlg-catalog/tables/other/rmpt01.html

**Comment by gmbecker** [comment]
Indeed, I think its a 'known issue' that some of the templates require changing. 

The only thing I can really say in terms of 'supporting' this is that it is possible to manually subset tables in both the row and column direction. You won't get any of the context-repetition, but in degenerate cases like you're describing that is actually "a feature".  That is essentially the best you'll be able to do for vertical pagination of a table like this.

So you'd do:

```
source("<rpkgroot>/tests/testthat/setup-fakedata.R")

tbl <- tt_to_export()

lpp <- 70

rowdf <- make_row_df(tbl)

rowexts_cuml <- cumsum(rowdf$self_extent)

part_is <- split(seq_len(NROW(rowdf)), cut(rowexts_cuml, c(seq(0, NROW(rowdf), by = lpp), nrow(rowdf))))

mypags <- lapply(part_is, function(ii) tbl[ii, keep_titles = TRUE, drop = FALSE, reindex_refs = FALSE])
```

As you can see there are some indentation issues there, but those are a function of subsetting rows in a non-structurally aware way, and may or may not be mitigatable.

One thing you could do is make the creation of the `part_is` there cleverer so that it breaks at structurally sound points instead of on raw row count, but I wrote the above quickly so I didn't do that here.

**Comment by clarkliming** [comment]
> One thing you could do is make the creation of the `part_is` there cleverer so that it breaks at structurally sound points instead of on raw row count, but I wrote the above quickly so I didn't do that here.

@shajoezhu do you think this function should be included in chevron/tern/rtables?

**Comment by gmbecker** [comment]
@clarkliming @shajoezhu this is much more work for less benefit than simply changing the templates to use analyze instead of summarize row groups. That is a pretty straightforward and narrow change.

This is a janky workaround hack that only half works and wont' be fully supported (ie we're not going to spend a bunch of time fixing the indenting stuff there, because its technically correct for the structure.

**Comment by Melkiades** [comment]
I will fix this today. This is not dependent on `formatters` nor `rtables` as it is an issue of the template itself and should be fixed in `tern` and then in the template. This happens because of `tern` masking the real rtables functions. I am closing this and opening a related issue in tern. I have a general solution to this.
